Core/Core/Assets.xcassets/trophy.imageset/Contents.json

```diff
@@ -0,0 +1,12 @@
+{
+  "images" : [
+    {
+      "filename" : "Regular-M.pdf",
```

Comment:
Could you please change the file name to the asset name?

Reply:
done